src/com/company/reverse/Reverse.java

```diff
@@ -0,0 +1,24 @@
+package com.company.reverse;
+
+
+public class Reverse {
+
+    static final String ALL_DIGITS_IN_STRING = "[\\d+]";
+    static final String STRING_REGEX = "";
+
+    public String reverse(String stringToReverse) {
```

Comment:
Please rename either this method or class so trey have different names. It’s a bit confusing

Reply:
Done